src/index.js

```diff
@@ -1,17 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
+import { Calculator, Display } from './Calculator';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
```

Comment:
- [ ] It would be better if we create an `App` component where we import the `calculator` component. Currently, the file has been deleted